crash task reobf with `kotlin-stdlib`

Current Workaround: exclude `META-INF` from embedding kotlin.

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':reobf'.<41 internal lines>
Caused by: java.lang.IllegalArgumentException
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at org.objectweb.asm.ClassReader.<init>(Unknown Source)
	at net.md_5.specialsource.JarRemapper.remapClassFile(JarRemapper.java:228)
	at net.md_5.specialsource.JarRemapper.remapJar(JarRemapper.java:189)
	at net.minecraftforge.gradle.tasks.user.reobf.ObfArtifact.applySpecialSource(ObfArtifact.java:385)
	at net.minecraftforge.gradle.tasks.user.reobf.ObfArtifact.generate(ObfArtifact.java:343)
	at net.minecraftforge.gradle.tasks.user.reobf.ReobfTask.doTask(ReobfTask.java:319)<38 internal lines>
	... 37 more
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -59,7 +59,8 @@ dependencies {
     compile(gradleApi())
 
     // moved to the beginning to be the overrider
-    compile("org.ow2.asm:asm-debug-all:5.0.3")
+    compile("org.ow2.asm:asm:9.1")
+    compile("org.ow2.asm:asm-tree:9.1")
     compile("com.google.guava:guava:18.0")
 
     compile("net.sf.opencsv:opencsv:2.3") // reading CSVs.. also used by SpecialSource

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -72,7 +72,7 @@ dependencies {
     compile("com.google.code.gson:gson:2.2.4") // Used instead of Argo for buuilding changelog.
     compile("com.github.tony19:named-regexp:0.2.3") // 1.7 Named regexp features
 
-    compile("net.md-5:SpecialSource:1.7.3") // deobf and reobs
+    compile("net.md-5:SpecialSource:1.9.0") // deobf and reobs
 
     // because curse
     compile("org.apache.httpcomponents:httpclient:4.5.13")

